plugins/notifications-backend/src/permissions.ts

```diff
@@ -0,0 +1,23 @@
+/* TODO(mlibra): move following to notifications-common for its easier reuse by frontend and policies */
```

Comment:
ok, I will create the notificaitons-common right away. Again.

Reply:
you can simply remove the comment. then optionally open a story. we could end up removing the common again so i am not sure it is worth your time.

Reply:
It's for a potential writer of the policies.
If we want to implement it properly, tt should be consumed by the FE as well to enable/disable the actions there. Maybe I will add it, just concentrating on a different issue now.

Reply:
please explain how the FE will use it.
all the rest are 'potential' and 'if we want'. again, future stuff that we should not add.

Reply:
i see 2 permissions: poster and reader

Reply:
We should support the case when FE plugin can send messages. Intentionally we do not have `Send` functionality on the NotificationsPage but a 3rd-party FE plugin calling the `create` of the Notifications plugin will further implement such check for the Create permission.
To do so, the 3rd-party plugin would better import a `-common` package instead of `-backend`.

I can imagine multiple other uses-cases which fit straight to the `notifications-frontend` but they are for later.

The permissions themselves are not a burden of the code, it's a few lines and keeps us closer to the architecture of the rest of the Backstage.

Reply:
Just for the record. The "less code" discussion is about a function with body of 13 lines more-or-less copied from the documentation.

Reply:
I have responded at the 2nd comment on that topic.